src/pages/home/index.tsx

```diff
@@ -50,7 +50,7 @@ function Flipboxes(props: FlipboxesProps): JSX.Element {
 
 export default function PageHome(): JSX.Element {
     // query site metadata for page content
-    const query = useStaticQuery(graphql`
+    const query = useStaticQuery<GatsbyTypes.HomeQueryQuery>(graphql`
```

Comment:
This can be removed now as the plugin to do this was uninstalled. They should no be auto added anymore